metals/src/main/scala/scala/meta/internal/metals/codeactions/ImportMissingSymbol.scala

```diff
@@ -43,14 +44,42 @@ class ImportMissingSymbol(compilers: Compilers) extends CodeAction {
         }
     }
 
+    def addImportAllMissingSymbol(
```

Comment:
```suggestion
    def importMissingSymbols(
```
Maybe this to better align with the other `importMissingSymbol` in the file.